src/kademlia/dht_tracker.cpp

```diff
@@ -393,7 +392,7 @@ namespace libtorrent { namespace dht
 		entry* n = (entry*)userdata;
 		std::string node;
 		std::back_insert_iterator<std::string> out(node);
-		write_endpoint(e.ep(), out);
+		libtorrent::detail::write_endpoint(e.ep(), out);
```

Comment:
is write_endpoint conflicting with something in winrt? an alternative could be to change the using statement to: using libtorrent::detail::write_endpoint;